tests/unit/IonBinaryWriterTest.js

```diff
@@ -185,30 +185,30 @@ define([
     writerTest('Writes null decimal by direct call',
       (writer) => { writer.writeNull(ion.TypeCodes.DECIMAL) },
         [0x5f]);
-    writerTest('Writes implicitly positive zero decimal as single byte',
-      (writer) => { writer.writeDecimal(ion.Decimal.parse("0")) },
-        [0x50]);
-    writerTest('Writes explicitly positive zero decimal as single byte',
-      (writer) => { writer.writeDecimal(ion.Decimal.parse("+0")) },
-        [0x50]);
-    writerTest('Writes negative zero decimal',
-      (writer) => { writer.writeDecimal(ion.Decimal.parse("-0")) },
-        [0x52, 0x80, 0x80]);
+    // this.skip();  writerTest('Writes implicitly positive zero decimal as single byte',
```

Comment:
What's the benefit of prepending `this.skip()` within each comment? Easier grepping?

As an alternative, I propose making an do-nothing function called `skippedWriterTest` and changing these calls from `writerTest` to `skippedWriterTest`. It'll be self-explanatory and the IDE can show you all of the skipped tests via 'find usages'.

Reply:
grepping was the plan, but I've updated to provide skipped* functions that can call `this.skip()`, so now skipped tests show up in intern's output, but don't fail the build